tests/e2e/README.md

```diff
@@ -0,0 +1,46 @@
+# Block SDK E2E Tests
+
+## Integrating/Running Tests In Your Project
+
+The Block SDK E2E tests are built around [interchaintest](https://github.com/strangelove-ventures/interchaintest). To run the 
+full set of predefined test cases using your chain you can follow the steps below:
+
+### Integrate the Block SDK into your project  
+
```

Comment:
nit: the e2e tests are exported as a separate dependency from the block-sdk, it doesn't seem clear here? Maybe a link to the e2e tests dependency would be useful>